pandas/io/json/json.py

```diff
@@ -473,7 +478,7 @@ def read_json(
 
     Returns
     -------
-    result : Series or DataFrame, depending on the value of `typ`.
+    Series or DataFrame, depending on the value of `typ`.
```

Comment:
Our standard is to leave just the type `Series or DataFrame` in this line, and in the next indented to add any explanation. If you can update here, that would be great.

Reply:
Will do.